.gitignore

```diff
@@ -30,6 +30,9 @@
 #rosdoc_lite
 /doc/
 
+#Speed_compare
```

Comment:
Might want to create a folder instead of just ignoring a single file. Then I can create multiple and store them there.